aspnetcore/release-notes/aspnetcore-9/includes/signalr-trimming-aot.md

```diff
@@ -0,0 +1,26 @@
+### SignalR supports trimming and native AOT
+
+Continuing the [native AOT journey](https://learn.microsoft.com/aspnet/core/fundamentals/native-aot) started in .NET 8, we have enabled trimming and native ahead-of-time (AOT) compilation support for both SignalR client and server scenarios. You can now take advantage of the performance benefits of using native AOT in applications that use SignalR for real-time web communications.
+
+#### Getting started
+
+Install the latest [.NET 9 SDK](https://get.dot.net/9).
+
+Create a solution from the `webapiaot` template in a command shell using the following command:
+
+```dotnetcli
+dotnet new webapiaot -o SignalRChatAOTExample
+```
+
+Replace the contents of the `Program.cs` file with the following SignalR code:
+
+[!code-csharp[](~/release-notes/aspnetcore-9/samples/SignalRChatAOTExample/Program.cs)]
+
+The preceding example produces a native Windows executable of 10 MB and a Linux executable of 10.9 MB.
+
+#### Limitations
```

Comment:
Can we just copy the content from https://github.com/dotnet/release-notes-drafts/edit/main/9.0/preview7/aspnetcore.md#limitations?